InscryptionAPI/Helpers/CustomLine.cs

```diff
@@ -0,0 +1,82 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using DiskCardGame;
+
+namespace InscryptionAPI.Helpers
+{
+    public class CustomLine
```

Comment:
What is the purpose of the CustomLine class? Why can't the user just use DialogueEvent.Line directly?

Reply:
i cant because i cant make convert operators for dialogueevent.line which are very useful when creating lines